API/Controllers/AccountController.cs

```diff
@@ -1,28 +1,26 @@
-using System.Runtime.CompilerServices;
+using System.Reflection.Metadata.Ecma335;
 using System.Security.Cryptography;
-using System.Text;
 using API.data;
 using API.DTOs;
 using API.Entities;
 using API.Interfaces;
 using AutoMapper;
-using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
-using Microsoft.EntityFrameworkCore.Diagnostics;
 
 namespace API.Controllers
 {
     public class AccountController : BaseApiController
     {
-        private readonly DataContext _context;
         private readonly ITokenService _tokenService;
         private readonly IMapper _mapper;
-        public AccountController(DataContext context, ITokenService tokenService, IMapper mapper)
+        private readonly UserManager<AppUser> _userManager;
+        public AccountController(UserManager<AppUser> userManager, ITokenService tokenService, IMapper mapper)
         {
+            _userManager = userManager;
```

Comment:
@August269 
On private class members you can use **`this`** keyword without adding that _ before variable names.

That would look like below: 
`this.userManager = userManager;`

Just for information... both ways are equally good and often used.